torch/_guards.py

```diff
@@ -843,7 +843,7 @@ def name(self) -> str:
     def make_guard(self, fn) -> Guard:
         if self.guard_source() is GuardSource.CONSTANT:
             raise NotImplementedError
-        return Guard(self, fn)
+        return Guard(originating_source=self, create_fn=fn)
```

Comment:
So @aorenste discovered that using named arguments is slower than passing them as positional, since guards are hot this might funny be a regression